### PR TITLE
Category mapping view v2

### PR DESCRIPTION
**Pull request by NileshPant1999**

### CODE REVIEW

File: fyle_accounting_mappings/models.py [hunk 1]
```diff
@@ -673,7 +673,7 @@ class CategoryMapping(models.Model):
     Category Mappings
     """
     id = models.AutoField(primary_key=True)
-    source_category = models.OneToOneField(ExpenseAttribute, on_delete=models.PROTECT)
+    source_category = models.ForeignKey(ExpenseAttribute, on_delete=models.PROTECT, related_name='categorymapping')
```
Comment (ashwin1111) [inline fyle_accounting_mappings/models.py:676]: this should be 1-1 field only, why is this changed to ForeignKey?

File: fyle_accounting_mappings/serializers.py [hunk 2]
```diff
@@ -265,6 +265,17 @@ def to_representation(self, data):
...
+    Serializer to filter the active system, which is a boolen field in
+    System Model. The value argument to to_representation() method is
+    the model instance
+    """
+
+    def to_representation(self, data):
+        return super(CategoryMappingFilteredListSerializer, self).to_representation(data)
+
+
 class EmployeeMappingSerializerV2(serializers.ModelSerializer):
```
Comment (ashwin1111) [inline fyle_accounting_mappings/serializers.py:279]: https://fylein.slack.com/archives/C053VAAAJU9/p1695192320117999?thread_ts=1694772691.922109&cid=C053VAAAJU9 please incorporate this change too, this is for adding card account. Just 1 thing, spin up qbo app and check if this addition breaks anything

**Review by anishfyle — APPROVED** [review]
